File: Getting Started - WCF RIA Services/[C#]-Getting Started - WCF RIA Services/C#/HRApp.Web/Services/AuthenticationService.cs
namespace HRApp.Web
{
    using System.Security.Authentication;
    using System.ServiceModel.DomainServices.Hosting;
    using System.ServiceModel.DomainServices.Server;
    using System.ServiceModel.DomainServices.Server.ApplicationServices;
    using System.Threading;

    /// <summary>
    ///    RIA Services DomainService responsible for authenticating users when
    ///    they try to log on to the application.
    ///    
    ///    Most of the functionality is already provided by the base class
    ///    AuthenticationBase
    /// </summary>
    [EnableClientAccess]
    public class AuthenticationService : AuthenticationBase<User> { }
}

File: Getting Started - WCF RIA Services/[VB]-Getting Started - WCF RIA Services/VB/HRApp.Web/Services/AuthenticationService.vb
Imports System.Security.Authentication
Imports System.ServiceModel.DomainServices.Hosting
Imports System.ServiceModel.DomainServices.Server
Imports System.ServiceModel.DomainServices.Server.ApplicationServices
Imports System.Threading
Namespace Web

    ''' <summary>
    ''' RIA Services DomainService responsible for authenticating users when
    ''' they try to log on to the application.
    ''' 
    ''' Most of the functionality is already provided by the base class
    ''' AuthenticationBase
    ''' </summary>
    <EnableClientAccess()> _
    Public Class AuthenticationService
        Inherits AuthenticationBase(Of User)
    End Class
End Namespace